Products/Zuul/routers/zep.py

```diff
@@ -648,6 +648,17 @@ def manage_events(self, evids=None, excludeIds=None, params=None, uid=None, asof
             return False
         manage_events_for = (r.managedObjectName() for r in user.getAllAdminRoles() if r.role in READ_WRITE_ROLES)
         return organizer_name in manage_events_for
+    
+    def add_events(self, summary, device, component, severity, evclasskey,
+                  evclass=None, monitor=None, **kwargs):
+        ctx = self.context.dmd.Devices.findDevice(device.strip())
+        if not ctx:
+            ctx = self.context
+
+        if Zuul.checkPermission(ZEN_MANAGE_EVENTS, ctx):
+            return True
+        else:
+            return False
```

Comment:
While using an `if` statement is explicit, I believe it is redundant because `Zuul.checkPermission` already returns a boolean (or boolean like) value. So the return can be written more succinctly as:
```
return Zuul.checkPermission(ZEN_MANAGE_EVENTS, ctx)
```

Reply:
Yeah, it is a little bit stupid